frontend/packages/core/src/NPS/feedback.tsx

```diff
@@ -0,0 +1,219 @@
+import React, { useState } from "react";
+import { clutch as IClutch } from "@clutch-sh/api";
+import styled from "@emotion/styled";
+import { Grid as MuiGrid } from "@material-ui/core";
+import MuiSuccessIcon from "@material-ui/icons/CheckCircle";
+import { debounce } from "lodash";
+import { v4 as uuid } from "uuid";
+
+import { userId } from "../AppLayout/user";
+import { Button } from "../button";
+import { Alert } from "../Feedback";
+import { TextField } from "../Input";
+import { client } from "../Network";
+import type { ClutchError } from "../Network/errors";
+import { Typography } from "../typography";
+
+import EmojiRatings, { Rating } from "./emojiRatings";
+
+/** Interfaces */
+
+interface FeedbackOptions {
+  origin: "ORIGIN_UNSPECIFIED" | "WIZARD" | "ANYTIME";
```

Comment:
```suggestion
  origin: "WIZARD" | "ANYTIME";
```
API doesn't allow for unspecified origins (but enums require that a 0 option always be set)

Reply:
Ohh, didn't realize that was how that worked, nice